python/cog/types.py

```diff
@@ -150,9 +149,11 @@ def __init__(self, *, source: str, filename: str, fileobj: io.IOBase) -> None:
 
     def convert(self) -> Path:
         if self._path is None:
-            dest = tempfile.NamedTemporaryFile(suffix=self.filename, delete=False)
-            shutil.copyfileobj(self.fileobj, dest)
-            self._path = Path(dest.name)
+            with tempfile.NamedTemporaryFile(
+                suffix=self.filename, delete=False
+            ) as dest:
+                shutil.copyfileobj(self.fileobj, dest)
+                self._path = Path(dest.name)
```

Comment:
Double-check that use of context manager `NamedTemporaryFile` with `delete=False` does what we expect.

Reply:
On review, I don't think this is equivalent. Backing out of this change.